tests/data/product_stability/alinux2.yml

```diff
@@ -9,10 +9,10 @@ benchmark_id: ALINUX-2
 benchmark_root: ../../linux_os/guide
 chrony_conf_path: /etc/chrony.conf
 cpes:
-- alinux2:
-    check_id: installed_OS_is_alinux2
-    name: cpe:/o:alinux:alibaba_cloud_linux:2
-    title: Alibaba Cloud Linux 2
+  - alinux2:
+      check_id: installed_OS_is_alinux2
+      name: cpe:/o:alinux:alibaba_cloud_linux:2
+      title: Alibaba Cloud Linux 2
```

Comment:
@svet-se  What is the reason of these whitespace-only changes?

Reply:
@jan-cerny it was generated by running "python ./tests/test_product_stability.py --update-reference-data ./products ./tests/data/product_stability" to update the test data

Reply:
Sounds like we either have to update the tool to generate files that do not disrupt existing files in terms of indentation or only the meaningful changes are kept in this PR (removing all unnecessary white spaces).

Reply:
It looks to me like a fix for test_product_stability.py, but for some reason reason the files with the new indentation are not regenerated. IMHO this indentation should be OK. @ggbecker please tell me your thoughts about this. There is also an option to add a parameter for a particular product, i.e. to regenerate the test files only for SLE in this case, not for all products in the directory (but in the particular case we caught 2 products without test files).

Reply:
I can confirm that when I run on my setup `python ./tests/test_product_stability.py --update-reference-data ./products ./tests/data/product_stability` I don't get the changes in the indentation so seems like the yaml generator , used in the test_product_stability tool has some environment dependency

Reply:
I've reverted the latest commit and just updated the sle15 test data

Reply:
@teacup-on-rockingchair it seems that the issue is related to the Python version, I could confirm that with 3.11 there is no issue and the format is correct.